newscript.go

```diff
@@ -290,26 +236,26 @@ func findVbutil() error{
 	return nil
 }
 
-func buildVbutil() error{
+func buildVbutil() error {
 	fmt.Printf("-------- Building in Vbutil\n")
-	cmd8 := exec.Command("git", "clone", "https://chromium.googlesource.com/chromiumos/platform/vboot_reference")
-	err := cmd8.Run()
+	cmd := exec.Command("git", "clone", "--depth", "1", "https://chromium.googlesource.com/chromiumos/platform/vboot_reference")
+	err := cmd.Run()
 	if err != nil {
 		fmt.Printf("didn't get chromium repo")
 		return err
 	}
 	return nil
-	cmd9 := exec.Command("git", "checkout", "3f3a496a23088731e4ab5654b02fbc13a6881c65")
-	err = cmd9.Run()
+	cmd = exec.Command("git", "checkout", "3f3a496a23088731e4ab5654b02fbc13a6881c65")
```

Comment:
In this one case, the -depth thing may cause trouble if you try to checkout a commit that is at a deeper depth ...

we'll see I guess!